### DISCUSSION TITLE
A warning/query on the GUI [solved]

### DISCUSSION THREAD

**Discussion by johnsidoli** [abraunegg/onedrive#2230] [General]
I just attempted to install and run the GUI linked here, after many months of successfully using the client without, and it seems to have 'wiped' my config and key.

It was probably a bit stupid of me but I presumed it would just run ontop of the existing config.

Now when I type 'onedrive' and try to do anything in the terminal I get the following error, and also onedrive (main sync dir) no longer works:

```
Unknown key in config file: force_http_11
Configuration file has errors - please check your configuration
```

My sharepoint still works though in another config.

Any ideas how I can fix my main onedrive and get it working again? QUickest route to delete everything and start over?

**Comment by johnsidoli** [comment]
Managed to get it back via a backup config that was there. Phew!

Not sure this GUI is quite ready for lo-tech users like me !

**Comment by abraunegg** [comment]
@johnsidoli 
> `Unknown key in config file: force_http_11`

All you need to do is remove 'force_http_11' from your config file. This was depreciated a *long time ago*

> I just attempted to install and run the GUI linked here

Issues with the GUI are *not* supported here. Please raise your issue and problem here: https://github.com/bpozdena/OneDriveGUI
